nix/package.nix

```diff
@@ -0,0 +1,556 @@
+    "extension"
+  ]));
+
+  python-monkey-business = simplePackage {
+    name = "python-monkey-business";
+    version = "1.0.0";
+    hash = "sha256-mXZSKYl2bwCyqqJOyW6suRpt57cAHRRSB5MjsHGYjg4=";
+
+    propagatedBuildInputs = [
+      six
+    ];
+  };
+  django-phonenumber-field = simplePackage {
+    name = "django-phonenumber-field";
+    version = "6.4.0";
+    hash = "sha256-cqPno+dJO/KhLAejvHfOiYE6zBZZK/BNDu47WkUgl+0=";
+
+    format = "pyproject";
+
+    nativeBuildInputs = [
+      setuptools
+      setuptools_scm
+    ];
+
+    propagatedBuildInputs = [
+      django
+    ];
+  };
+  django-selenosis = simplePackage {
+    name = "django-selenosis";
+    version = "2.0.0";
+    hash = "sha256-/MSC5/yAv+dN7JwrY0LY2R45gI0ub48Ldg05TKqmubE=";
+
+    doCheck = false;
+
+    propagatedBuildInputs = [
+      django
+    ];
+  };
+  o-vine = overridePackage vine {
+    version = "1.3.0";
+    hash = "sha256-Ez7m16kBbxd93q8ZHB9YQhodzG7ppCxYs0vtQOHSzYc=";
+
+    pythonImportsCheck = [ "vine" "vine.five" ];
+  };
+  o-billiard = overridePackage billiard {
+    version = "3.6.4.0";
+    hash = "sha256-KZ3lqNoop4PVGxl9SWvvTxWV3QI6k6T1nd4Yhq6QVUc=";
+  };
+  o-pluggy = overridePackage pluggy {
+    version = "0.13.1";
+    hash = "sha256-FbKs3mZlYeEpjXG1IwB+1zZN4HApIZtgTPgIv6HHZbA=";
+  };
+  amqp = simplePackage {
+    name = "amqp";
+    version = "2.6.1";
+    hash = "sha256-cM2xBihGj/FOV+wvdRx6qeSOfjZRz9YtQxITwMTljyE=";
+
+    checkPhase = "";
+    doCheck = false;
+    postPatch = ''
+      rm requirements/test.txt
+      sed -i "s/reqs('test.txt')/[]/" setup.py
+    '';
+
+    propagatedBuildInputs = [ o-vine ];
+  };
+  o-kombu = overridePackage kombu {
+    version = "4.6.11";
+    hash = "sha256-yhtF+qyMCxhJPQKoVxeS88QCkc8rzx9Vr+09jzqnunQ=";
+
+    disabledTestPaths = [
+      "t/unit/transport/test_SQS.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_azureservicebus.py"
+      "t/unit/transport/test_filesystem.py"
+      "t/unit/transport/test_filesystem.py"
+    ];
+
+    propagatedBuildInputs = [ amqp ];
+  };
+  o-watchdog = overridePackage watchdog {
+    propagatedBuildInputsFilter = x: (lib.getName x) == "PyYAML";
+    extraPropagatedBuildInputs = [ o-pyyaml ];
+  };
+
+  # o-pytz = overridePackage pytz {
+  #   version = "2021.1";
+  #   hash = "sha256-g6SpCJS/OOJDzwUsi1jzgb/pp6SD9qnKsUC8f3AqxNo=";
+  # };
+  o-pyyaml = overridePackage pyyaml {
+    pypiName = "PyYAML";
+    version = "5.4.1";
+    hash = "sha256-YHd0y7oocyv6gCtUuqdIQhX1MJkQVbtWLvvtWy8gpF4=";
+
+    postPatch = "rm -r tests/lib{,3}";
+    checkPhase = "";
+    installCheckPhase = "";
+    doCheck = false;
+    doInstallCheck = false;
+    patches = [ ./pyyaml-disable-tests.patch ];
```

Comment:
Is ALL of this needed?

Reply:
Most likely.
I think the only way to check is to research whether there were any breaking changes in these packages and update to the nixpkgs versions or by updating, testing and replacing everything that breaks.

Reply:
Let's leave this for the 23.05 migration then.
~~we'll see whether it will really be .05 and not .11~~